tests/server/blueprints/cases/test_matchmaker_controllers.py

```diff
@@ -59,21 +59,11 @@ def test_mme_add(self, populated_database, user_obj, case_obj, parsed_variant):
         session = requests.Session()
         adapter = populated_database
 
-        # Add an HPO term to this scout case
-        assert case_obj.get("phenotype_terms") is None
-        phenotype_term = {
-            "phenotype_id": "HP:0011031",
-            "feature": "Abnormality of iron homeostasis",
-        }
-        updated_case = adapter.case_collection.find_one_and_update(
-            {"_id": case_obj["_id"]},
-            {"$set": {"phenotype_terms": [phenotype_term]}},
-            return_document=pymongo.ReturnDocument.AFTER,
-        )
-        assert updated_case["phenotype_terms"][0] == phenotype_term
+        # Given a case with  HPO terms
+        assert case_obj.get("phenotype_terms")
```

Comment:
This is already present in the case object now, so no need to add phenotypes before